platypus/parse.py

```diff
@@ -263,20 +270,19 @@ def process_results(percentage_ids, alignment_lengths, percentage_ids_other,
             # Comparing bit_scores to create outputs
             if vals['a']['bit_score'] == vals['b']['bit_score']:
                 results[i]['equal'] += 1
-                results[i]['summary'].append('%s\t%s\t%s' % (seq_name,
-                                                             subject_id_a,
-                                                             subject_id_b))
+                results[i]['summary_fh'].write('%s\t%s\t%s\n' % (
+                    seq_name, subject_id_a, subject_id_b))
                 db_seqs_counts_a[subject_id_a] += 1
                 db_seqs_counts_b[subject_id_b] += 1
             elif vals['a']['bit_score'] > vals['b']['bit_score']:
                 if not subject_id_b:
                     results[i]['perfect_interest'] += 1
-                    results[i]['summary'].append('%s\t%s\t' % (seq_name,
-                                                               subject_id_a))
+                    results[i]['summary_fh'].write('%s\t%s\t\n' % (
```

Comment:
You are right, it expects 3 values: seq_id best_A best_B. In this case, there is no best_B. Thus, just adding a new line is fine cause there is no value there. However, I can add something more specific but not sure what.

The results shouldn't be jagged. BTW there is a test that checks that the resulting files are the same.